src/bundle.ts

```diff
@@ -17,21 +18,35 @@ const _bundle = (
   cur?: unknown,
   bundledRefInventory: any = {},
   bundledObj: any = {},
+  errorsObj: any = {},
 ) => {
   const objectToBundle = get(document, pointerToPath(path));
```

Comment:
@lottamus this is where the error [invalid JSON pointer error ](https://sentry.io/organizations/stoplightio/issues/1766465039/events/latest/?project=5257888&referrer=github_integration) was coming from. So errors in thrown in pointerToPath won't make it into the errorsObj, right?

Reply:
@chohmann and I chatted on slack and decided it's ok to throw here. If an invalid pointer is passed, we won't know the path to the object we're trying to bundle and should fail anyway.